networks/3_module_validation.py

```diff
@@ -113,9 +123,14 @@ def plot_dist_modules(clustering_method_list):
 plot_dist_modules(clustering_method_list)
 
 # **Takeaway:**
-# * Looks like there is one large modules using DBSCAN and hierarachal clustering
-# * There are more even sized modules using affinity propogation
-# * For all methods, the size of most modules are 10-50's (Note: as a results of the binning which is summing counts across multiple modules, the height of the binds are in the hundreds)
+# Our expectation on size of modules would be 2-50 genes. Most operons have fewer than 10 genes and most regulons have fewer than 100 genes. Some examples that demonstrate the size of co-expression networks can be found in papers using ADAGE signatures to define modules:
+# * Figure 5 in [eADAGE paper](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1905-4.pdf)
+# * Figure 7 in [Harty et al. paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6531624/)
+# * Figure 2 in [Doing et al. paper](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008783)
+#
+# What did we find? Which method follows our expectation? This is the one we will move forward.
+# * Looks like there is one large modules using DBSCAN clustering
```

Comment:
Did you try to vary the parameters of DBSCAN (eps and min_samples) and see if you find module sizes as you expect? Maybe that could be another way of starting these analyses: if you already know what module sizes should be, then for each algorithm you try you could change parameters until finding something you expect.

Reply:
I did play around with the parameters somewhat, but I can try some more and see what happens